tests/compas/utilities/test_async_.py

```diff
@@ -1,112 +0,0 @@
-from threading import Thread
```

Comment:
It took me a bit to realize that these test file was duplicate and that's the reason for deletion here :)

Reply:
ha sorry, yes, i should have probably mentioned that. not sure why these duplicates existed...